app/routes/extended-access.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import express from 'express';
+import bodyParser from 'body-parser';
+
+const router = express.Router();
+/**
+ * GET /api/extended-access/cookie/:cookieName
+ * A getter method that returns the current httpOnly cookie.
+ * This can be done client-side too, and is here for consistency.
+ */
+router.get('/cookie/:cookieName', async (req, res) =>{
```

Comment:
@ed-bird added new `GET` and `POST` methods for handling httpOnly cookies.

Reply:
Thanks for adding!